transform/mattermost-analytics/models/staging/mm_telemetry_rc/_mm_telemetry_rc__models.yml

```diff
@@ -21,4 +21,75 @@ models:
       - name: server_id
         description: The ID of the server the event originated from.
       - name: received_at
-        description: Timestamp registered by RudderStack when the event was ingested (received).
\ No newline at end of file
+        description: Timestamp registered by RudderStack when the event was ingested (received).
+
+        
+  - name: stg_mm_telemetry_rc__performance_events
+    description: |
+      Reconstructed `performance_events` table using only columns needed by looker.
```

Comment:
Maybe document/mention the difference between `rc` and prod`, otherwise the descriptions are identical.